Cleanup development files.

mike820324 opened the pull request:
- remove main.py
- move application.cfg and log.cfg to conf folder.

After this commit, in test environment use 

``` bash
make install && mpserver --config-file ./conf/application.cfg
```

This PR depends on #229, since the logger config file is moved to conf folder.

mike820324 committed:
Cleanup development files.

- remove main.py
- move application.cfg and log.cfg to conf folder.

chhsiao90 commented:
Is there anyway that I could run mpserver without define config path after this PR?

mike820324 commented:
Without specified the options, the only way maybe cd into the conf folder.

Or maybe adding a new entry in Makefile

``` Makefile
run-server:
    mpserver --config-file conf/application.cfg
```

This is a good point. It may bring a lot of inconvenience when developing right?
I'll modify this PR a bit, only removing the main.py, but remain the configuration files.

chhsiao90 commented:
I can accept with the Makefile solution if there is no better solution!

mike820324 committed:
Add new entry in Makefile.

Add run-server entry in Makefile, which will point to the main config
location.
